src/controllers/words.js

```diff
@@ -64,8 +69,21 @@ export const getWords = async (req, res, next) => {
       dialects,
       examples,
     };
-    let query = !strict ? searchIgboTextSearch(searchWord, isUsingMainKey) : strictSearchIgboQuery(searchWord);
-    const words = await searchWordUsingIgbo({ query, ...searchQueries });
+    let query = !strict
+      ? searchIgboTextSearch(
+        searchWord
+          .split('')
+          .filter((c) => c !== '"' && c !== "'" && !(searchWord.match(/'.*'/) || searchWord.match(/".*"/)))
+          .join(''),
+        isUsingMainKey,
+      )
+      : strictSearchIgboQuery(
+        searchWord
+          .split('')
+          .filter((c) => c !== '"' && c !== "'" && !(searchWord.match(/'.*'/) || searchWord.match(/".*"/)))
+          .join(''),
```

Comment:
This might be more complicated than it needs to be! From the looks of it, you want to remove the quotes from the `searchWord`
I don't think we need these function calls if we are searching with Igbo since Igbo queries should not have quotes in the first place, because in order to get to this part of the function the keyword must not have any quotes.